### PR TITLE
mpssh 1.3.3

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/mpssh.rb [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+  homepage "https://github.com/ndenev/mpssh"
+  url "https://github.com/ndenev/mpssh/archive/1.3.3.tar.gz"
+  sha1 "ba11dfe7607cac3d47f1c86db236a2e440700ce7"
 
   def install
-    system "make", "CC=#{ENV.cc}"
-    bin.install "mpssh"
-    man1.install "mpssh.man" => "mpssh.1"
+    # don't install binaries as root
+    inreplace "Makefile", "-o root", ""
```
Comment (mikemcquaid) [inline Library/Formula/mpssh.rb]: Can you submit a patch upstream for this? Thanks!

Comment (bfontaine) [inline Library/Formula/mpssh.rb]: Yep, done.

File: Library/Formula/mpssh.rb [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
-  url 'https://github.com/ndenev/mpssh/archive/1.3.1.tar.gz'
-  sha1 'c9ae08c1449962a0585e9cbf5e6174581912a053'
+  homepage "https://github.com/ndenev/mpssh"
+  url "https://github.com/ndenev/mpssh/archive/1.3.3.tar.gz"
+  sha1 "ba11dfe7607cac3d47f1c86db236a2e440700ce7"
+
+  patch do
+    # don't install binaries as root
+    # upstream pull request: https://github.com/ndenev/mpssh/pull/22
+    url "https://github.com/bfontaine/mpssh/commit/3cbb868b6fdf8dff9ab86868510c0455ad1ec1b3.patch"
```
Comment (mikemcquaid) [inline Library/Formula/mpssh.rb]: Use `.digg` here instead.

Comment (bfontaine) [inline Library/Formula/mpssh.rb]: @mikemcquaid fixed.

File: Library/Formula/mpssh.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/mpssh.rb]: Update: The patch has been merged in HEAD.

**Comment by mikemcquaid** [comment]
`sha1` looks wrong: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20498/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_mpssh/

**Comment by bfontaine** [comment]
Thanks, fixed.

**Pull request closed without merging**